prov/rxm/src/rxm.h

```diff
@@ -74,7 +74,7 @@ extern size_t rxm_eager_limit;
 
 #define RXM_IOV_LIMIT 4
 
-#define RXM_EQ_PROGRESS_SPIN_COUNT 10000
+#define RXM_POLL_FREQ_USEC 10000
```

Comment:
No need to define separate macro. Variable `rxm_poll_freq_usec` can be set to 10000 and be overridden if the user requests a different value through env variable.